docs/hosted.rst

```diff
@@ -0,0 +1,53 @@
+Hosted Kingfisher
+=================
+
+Open Contracting Partnership operates a hosted instance of the Kingfisher tool suite for the use of OCP staff and OCDS Team members. If you think you should have access to this, contact `<mailto:code@opendataservices.coop>`_.
+
+Access
+------
+
+The server can be found at 195.201.163.242. You can connect via SSH to run commands, or via a Postgres client to run queries on the database (see below)
+
+There is a development server at ocdskingfisher-dev.default.opendataservices.uk0.bigv.io. If you're involved in the development of new Kingfisher components, you may be asked to log into the dev server to try things out. If so, substitiute the address of the dev server for the address of the live server in the examples below. 
```

Comment:
Technically this does not cover all the details for how to access dev server; http://scrape.ocdskingfisher.opendataservices.coop and http://scrape.ocdskingfisher-dev.default.opendataservices.uk0.bigv.io need to be listed.

Reply:
I've removed the specifics from the doc. I figure that if we're involving people in development, we'll tell them where the dev servers are, right?